src/settings/Settings.tsx

```diff
@@ -238,7 +239,26 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         controller: new ReloadOnChangeController(),
         displayName: _td("Threaded messaging"),
         supportedLevels: LEVELS_FEATURE,
-        default: false,
+        default: true,
+        betaInfo: {
+            title: _td("Threads"),
+            caption: () => <>
+                <p>{ _t("Keep discussions organised with threads.") }</p>
+                <p>{ _t("Threads help keep conversations on-topic and easy to track.") }</p>
+            </>,
+            disclaimer: () => <>
+                { SdkConfig.get().bug_report_endpoint_url && <>
+                    <h4>{ _t("How can I start a thread?") }</h4>
+                    <p>{ _t("Use “Reply in thread” when hovering over a message.") }</p>
```

Comment:
```suggestion
                    <p>{ _t("Use "Reply in thread" when hovering over a message.") }</p>
```

Reply:
they don't translate well, so in the past we've always used regular quotes to communicate intent to translators ftr

Reply:
@gsouquet @turt2live Ideally, we should support typographical quotes in localizable strings and let translators use the correct quotation marks for their locale. That said, we'd need a handbook/styleguide for translator and I don't even know if we have one.

https://en.wikipedia.org/wiki/Quotation_mark#Summary_table